src/app/server/Mdns.h

```diff
@@ -24,21 +24,38 @@ namespace chip {
 namespace app {
 namespace Mdns {
 
+enum class CommissioningMode
+{
+    kDisabled,                        // Commissioning Mode is disabled, CM=0, AC=0 in DNS-SD key/value pairs
+    kEnabled,                         // Basic Commissioning Mode, CM=1, AC=0 in DNS-SD key/value pairs
```

Comment:
nit: consider kBasic and kEnhanced as constants to match the comments.

Reply:
done